docs/new_labels.md

```diff
@@ -0,0 +1,27 @@
+# Adding New Labels
+New Docker Labels may be created and/or updated for the output image via the image metadata file.
+
+If a new label is specified in the metadata file, the label will be added in the output image.  However, any label previously defined in the base builder image will be ***overwritten*** in the output image, if the same label name is specified in the image metadata file.
+
+## Image Metadata File Name and Path
+The name and path of the file ***must*** be the following:
+```bash
+/opt/docker/image_metadata.json
```

Comment:
let's move this path to /tmp/docker/image_metadata.json.  that's the only path that's pretty much guaranteed to exist in any image and be writable by the assemble script.